cmd/healthbot/counterprobe/counterprobe.go

```diff
@@ -4,17 +4,14 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"net/url"
 	"sync"
 	"time"
 
 	"github.com/rs/zerolog"
 	logger "github.com/rs/zerolog/log"
-	"github.com/textileio/go-tableland/internal/router/rpcservice"
+	"github.com/textileio/go-tableland/pkg/client"
```

Comment:
All the changes is this file are to move healthbot from using raw json rpc calls to using the Go client.